packages/api-explorer/src/components/SideNav/SideNavMethods.tsx

```diff
@@ -44,19 +45,24 @@ export const SideNavMethods: FC<MethodsProps> = ({ methods, tag, specKey }) => {
     searchSettings: { pattern },
   } = useContext(SearchContext)
 
-  // TODO: make selected item visible
   return (
     <ul>
       {Object.values(methods).map((method) => (
         <li key={method.name}>
           <NavLink to={buildMethodPath(specKey, tag, method.name)}>
             <SidebarItem key={method.name} as="span">
-              <>
-                <MethodBadge verb={method.httpMethod} />
-                <Heading as="h5" truncate>
+              <Space gap="xsmall">
+                <MethodBadge
+                  alignTextCenter
+                  compact
+                  httpMethod={method.httpMethod}
+                >
+                  {method.httpMethod.toUpperCase()}
+                </MethodBadge>
+                <ApixHeading as="h5" mb="0" pt="0" fontWeight="light" truncate>
                   {highlightHTML(pattern, method.summary)}
-                </Heading>
-              </>
+                </ApixHeading>
```

Comment:
Now using the nicer names for endpoint and making use of the `truncate` to trim longer endpoint summaries.

Reply:
@josephaxisa per the other PR we want to look at adding tooltips to only truncated items.

Reply:
I'll address this in another PR.